confd/templates/base-storm-topology/topology.properties.tmpl

```diff
@@ -72,6 +72,10 @@ logger.level = INFO
 #######
 # Discovery
 # - discovery.interval = how many bolt ticks(1 tick per second) between ISL discovery / health checks, starts from 0
+# - discovery.slow.poll.interval = how many ticks (1 tick per second) between sending a discovery packet in case
```

Comment:
I propose the following poll mode names:
* generic - current discovery mode (minimal delay)
* exhausted - (no discovery response for N requests)
* auxiliary - (link health provided by some other than discovery poll mechanism)

The problem with slow and unhurried terms - they both define relative time units, but you can't guess which one of them is bigger. Because we must not make any promises about which one of these delays must be bigger, we should avoid any time unit (especially relative) in mode names.

PS This is just a proposition.
PSS There are no "ticks" into network topology now. Now it is used time-based units everywhere.

Reply:
I like your proposal. Names have been fixed.